src/e2e-test/java/io/cdap/plugin/gcsmove/stepsdesign/GCSMove.java

```diff
@@ -0,0 +1,207 @@
+    }
+
+    @Then("Enter GCSMove property location {string}")
+    public void enterGCSMovePropertyLocation(String location) {
+        GCSMoveActions.enterLocation(PluginPropertyUtils.pluginProp(location));
+    }
+
+    @Then("Close GCSMove properties")
+    public void closeGCSMoveProperties() {
+        CdfStudioActions.clickCloseButton();
+    }
+
+    @Then("Verify GCS Move property {string} invalid bucket name error message is displayed for bucket {string}")
+    public void verifyGCSMovePropertyInvalidBucketNameErrorMessageIsDisplayedForBucket(String property,
+                                                                                       String bucketName) {
+        CdfStudioActions.clickValidateButton();
+        String expectedErrorMessage = PluginPropertyUtils.errorProp(E2ETestConstants.ERROR_MSG_GCS_INVALID_BUCKET_NAME)
+                .replace("BUCKET_NAME", "/" + PluginPropertyUtils.pluginProp(bucketName));
+        String actualErrorMessage = PluginPropertyUtils.findPropertyErrorElement(property).getText();
+        Assert.assertEquals(expectedErrorMessage, actualErrorMessage);
+    }
+
+    @Then("Validate GCSMove successfully moved object {string} to destination bucket")
+    public void validateGCSMoveSuccessfullyMovedObjectToDestinationBucket(String path) throws IOException {
+        String sourceGCSBucket = TestSetupHooks.gcsSourceBucketName;
+        String gcsObject = PluginPropertyUtils.pluginProp(path);
+        for (Blob blob : StorageClient.listObjects(sourceGCSBucket).iterateAll()) {
+            if (blob.getName().contains(gcsObject)) {
+                Assert.fail("Object is not moved from source bucket" + sourceGCSBucket);
+                return;
+            }
+        }
+        BeforeActions.scenario.write("Object is moved from source GCS Bucket " + sourceGCSBucket + " successfully");
+        String targetGCSBucket = TestSetupHooks.gcsTargetBucketName;
+        try {
+            for (Blob blob : StorageClient.listObjects(targetGCSBucket).iterateAll()) {
+                if (blob.getName().contains(gcsObject)) {
+                    BeforeActions.scenario.write("Object moved to gcs bucket " + targetGCSBucket + " successfully");
+                    return;
+                }
+            }
+            Assert.fail("Object not moved to target gcs bucket " + targetGCSBucket);
+        } catch (StorageException | IOException e) {
+            if (e.getMessage().contains("The specified bucket does not exist")) {
+                Assert.fail("Target gcs bucket " + targetGCSBucket + " not created - " + e.getMessage());
+            } else {
+                Assert.fail(e.getMessage());
+            }
+        }
+    }
+
+    @Then("Validate GCSMove failed to move object {string} to destination bucket")
+    public void validateGCSMoveFailedToMoveObjectToDestinationBucket(String path) throws IOException {
+        String sourceGCSBucket = TestSetupHooks.gcsSourceBucketName;
```

Comment:
indent is incorrect. please reformat all files in the PR using these settings - https://cdap.atlassian.net/wiki/spaces/CE/pages/1594098619/Coding+Standard

Reply:
Reformat all files done